src/coreclr/jit/lclvars.cpp

```diff
@@ -858,9 +884,72 @@ void Compiler::lvaInitUserArgs(InitVarDscInfo* varDscInfo, unsigned skipArgs, un
             canPassArgInRegisters = varDscInfo->canEnreg(TYP_I_IMPL, cSlotsToEnregister);
         }
         else
+#elif defined(TARGET_LOONGARCH64)
+        var_types arg1_Type = TYP_UNKNOWN;
```

Comment:
We try not to use underscores in the names of locals (see https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/clr-jit-coding-conventions.md#8.4).

```suggestion
        var_types arg1Type = TYP_UNKNOWN;
```